src/index.html

```diff
@@ -9,6 +12,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="header">
+      <img class="img" src="images/logo.png"
```

Comment:
class `img` is not very informative
choose a better class that will say what this is an element and why it is needed